app/src/components/ProtvistaMSA.jsx

```diff
@@ -74,19 +78,49 @@ const ProtvistaMSAWrapper = () => {
     addLog(`[setColorScheme]: ${event.target.value}`);
   };
   const labelWidth = 100;
+  const conervationOptions = {
+    "calculate-conservation": true,
+  };
+  if (overlayConservation) {
+    conervationOptions["overlay-conservation"] = true;
+  }
+  if (sampleSizeConservation > 0) {
+    conervationOptions["sample-size-conservation"] = sampleSizeConservation;
+  }
 
   return (
     <>
       <h1>protvista-msa</h1>
-      <select
-        value={ColorScheme}
-        onChange={handleColorChange}
-        onBlur={handleColorChange}
-      >
-        {AllowedColorschemes.map((c) => (
-          <option key={c}>{c}</option>
-        ))}
-      </select>
+      <div style={{ display: "flex", flexDirection: "column" }}>
+        <label>
+          colorScheme:
+          <select
+            value={ColorScheme}
```

Comment:
Minor comment: `colorScheme` fits with the camelCase convention used for other variables.